backend/src/test/java/gov/cdc/usds/simplereport/api/converter/FhirConverterTest.java

```diff
@@ -804,7 +840,7 @@ void convertToDiagnosticReport_TestEvent_matchesJson() throws IOException {
                 getClass().getClassLoader().getResourceAsStream("fhir/diagnosticReport.json")),
             StandardCharsets.UTF_8);
 
-    JSONAssert.assertEquals(actualSerialized, expectedSerialized, true);
+    JSONAssert.assertEquals(expectedSerialized, actualSerialized, true);
```

Comment:
I didn't actually notice this until today, but the values were in the wrong order. It should go (expected, actual)